### PR TITLE
Aws taks 2

### PR DESCRIPTION
**Pull request by Stanislav-Berezovsky**
1. Task: https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md
- CloudFront URL (Automated: - https://d6rkhqqm91g07.cloudfront.net/
- S3 Bucket & CloudFront URLs (manual) - https://d2gb0uzqrzzczx.cloudfront.net/
- S3 Bucket(manual) replaced behaviour from the previous point - http://bsv-my-first-app.s3-website-eu-west-1.amazonaws.com/ 

2. Done 22.10.2020 / deadline 24.10.2020
3. Score: 5 / 5

### CODE REVIEW

**Review by petr150894 — APPROVED** [review]
Well done!

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+# you can add CloudFormation resource templates here
+resources:
+  Resources:
+    ## Specifying the S3 Bucket
+    WebAppS3Bucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ${self:custom.s3BucketName}
+        AccessControl: PublicRead
```
Comment (ezhukauskas) [inline serverless.yml:135]: I tested without AccessControl: PublicRead and it works fine. Actually I don't know why this option was added on the stream - CloudFront distribution works without it. Do you know? :)

**Review by ezhukauskas — APPROVED** [review]
Good job!

**Comment by azatdavliatshin** [comment]
Nice job!

**Review by azatdavliatshin — APPROVED** [review]

**Pull request merged**